functorch/_src/vmap.py

```diff
@@ -22,6 +23,23 @@
 in_dims_t = Union[int, Tuple]
 out_dims_t = Union[int, Tuple[int, ...]]
 
+
+def register_namedtuple_return_types():
```

Comment:
nit: torch.return_types are not exactly namedtuples :(. They have different semantics from Python namedtuple but they behave the same; the right terminology is Struct Sequence (https://docs.python.org/3/c-api/tuple.html#struct-sequence-objects).

Let's name this something like "register_torch_return_types()"?